graph.py

```diff
@@ -115,85 +114,94 @@ def incidence(self) -> List["Edge"]:
 
     @property
     def neighbours(self) -> ["Vertex"]:
-        """
-        Returns the list of neighbors of the vertex.
-        """
+        """Get the list of neighbours of the vertex."""
+
         return list(self._incidence.keys())
 
     @property
     def degree(self) -> int:
-        """
-        Returns the degree of the vertex
-        """
+        """Get the degree of the vertex."""
+
         return sum(map(len, self._incidence.values()))
 
+    @property
+    def label(self):
+        return self._label
+
+    @label.setter
```

Comment:
Wauw is dit een ding in python?

Reply:
Jep. Shortcut in PyCharm: `prop` (property getter) of `props` (property getter and setter).